lib/manageiq/appliance_console/certificate.rb

```diff
@@ -111,6 +115,23 @@ def enable_certmonger
 
     private
 
+    def save_self_signed
+      FileUtils.mv(key_filename, "#{key_filename}.ORIG", :force => true)  unless File.exist?("#{key_filename}.ORIG")
+      FileUtils.mv(cert_filename, "#{cert_filename}.ORIG", :force => true)  unless File.exist?("#{cert_filename}.ORIG")
+    end
+
+    def undo_tracking
+      stop_tracking
+      FileUtils.rm_f(root_filename) if File.exist?(root_filename)
```

Comment:
Should this file be saved in `save_self_signed` as well?

Reply:
The `root_filename` file is created by `getcert request` successfully getting signed certs
from the CA and does not exist when only the self signed certs are there.